tests/unit/test_replay_manager.py

```diff
@@ -84,7 +85,7 @@ def ringtest_virtual_pop_config():
 
 @pytest.mark.forked
 def test_sonata_spikes_reader():
-    from neurodamus.replay import SpikeManager, MissingSpikesPopulationError
+    from neurodamus.replay import MissingSpikesPopulationError, SpikeManager
```

Comment:
why do these need to be imported in a local function?  Is there state associated with them?

Reply:
Because they are used for this test. Do you prefer to import at the module level ?

Reply:
It's the general consensus that they should be at the module level, instead of a function (ie: in pep8 and pylint both flag them), unless there is a circular import, or other reason (which would probably have a comment attached)

Maybe this is one of the things that we can do in another PR, globally, but it would make life easier if we didn't add to it.

Reply:
ok, I've moved all the imports at the module level except the NEURON wrapper.

@juanjosegarcan  @cattabiani , please be aware of that in your ongoing/future PRs. In prinple, all imports should be at the module level **except**  
```
    from neurodamus.core import NeuronWrapper as Nd
```
As we run all the tests in forked processes,  import of `from neurodamus import Neurodamus` can be done at the module level too.

Reply:
thanks!

Reply:
Thank you for pointing it out @mgeplf . 
We didn't force that rule for the test files before, as we saw the strange neuron behaviors over multiple tests even in forked processes. But now it gets clear that we just need to import the neuron wrapper at function level.

Reply:
> But now it gets clear that we just need to import the neuron wrapper at function level.

That's something I'm hoping to find time to look at, so we could at least make that safe and not have to worry about strange behaviors.